WinUI3/AK.Toolkit.WinUI3.Localization/Localizer.cs

```diff
@@ -15,6 +15,53 @@ public partial class Localizer : DependencyObject, ILocalizer
 
     public IEnumerable<string> GetAvailableLanguages() => _languageResources.Keys;
 
+    private static Localizer Instance { get; set; } = null;
+
+    /// <summary>
+    /// Create a Instance of Localizer only once with Auto Register Elements
+    /// </summary>
+    /// <param name="resourcesFolderPath"></param>
+    /// <param name="resourcesFileName">Resources.resw</param>
+    /// <param name="defaultLanguage">en-US</param>
+    /// <returns></returns>
+    public static Localizer GetCurrent(string resourcesFolderPath, string resourcesFileName = "Resources.resw", string defaultLanguage = "en-US")
+    {
+        if (Instance == null)
+        {
+            Instance = new Localizer();
+            Instance.RunLocalizationOnRegisteredRootElements();
```

Comment:
You need to initialize first and then call ``RunLocalizationOnRegisteredRootElements()``.

Reply:
done